web-client/src/presenter/actions/TrialSessionWorkingCopy/getFormattedTrialSessionCasesAction.js

```diff
@@ -0,0 +1,14 @@
+import { state } from 'cerebral';
+
+/**
+ * getFormattedTrialSessionCasesAction
+ *
+ * @param {object} providers.get the cerebral get function
+ * @returns {Object} formattedCases
+ */
+
+export const getFormattedTrialSessionCasesAction = ({ get }) => {
+  const formattedCases =
+    get(state.trialSessionWorkingCopyHelper).formattedCases || [];
```

Comment:
Does this need to be defaulted to [] since the helper also seems to be defaulting it to []? i might be wrong, not sure